docs/content/en/docs/Architecture/network.md

```diff
@@ -2,24 +2,115 @@
+
+The coordination process in Kairos is designed to be resilient and self-coordinated, with no need for complex network configurations or control management interfaces. By using this approach, Kairos simplifies the process of deploying and managing Kubernetes clusters at the edge, making it easy for users to focus on running and scaling their applications.
+
+<p align="center">
+<img width="700" src="https://mudler.github.io/edgevpn/docs/concepts/architecture/edevpn_bootstrap_hu8e61a09dccbf3a67bf1fc604ae4924fd_64246_1200x550_fit_catmullrom_3.png">
+</p>
+
+### Packet flow
+
+The Virtual Private Network used is [EdgeVPN](https://github.com/mudler/edgevpn), which leverages [libp2p](https://github.com/libp2p/go-libp2p) for the transport layer.
+
+To explain how the packet flow works between two nodes, Node A and Node B, refer to the diagram below:
+
+<p align="center">
+<img width="700" src="https://user-images.githubusercontent.com/2420543/219048445-300de7e8-428f-4ded-848d-bf73c56acca1.png">
+</p>
+
+While partecipating actively to a network, each node keeps the shared ledger up-to-date with information about itself and how to be reachable by advertizing its own IP and the libp2p identity, allowing nodes to discover each other and know how to route packets. Assuming that we want to establish an SSH connection from Node A to Node B through the VPN network, which exposes the `sshd` service, the process is as follows:
+
+1. Node A (`10.1.0.1`) uses `ssh` to dial the VirtualIP of the Node B (`10.1.0.2`) in the network.
+2. EdgeVPN reads the frame from the TUN interface.
+3. If EdgeVPN finds a match in the ledger between the VirtualIP and an associated Identity, it opens a p2p stream to Node B using the libp2p Identity.
+4. Node B receives the incoming p2p stream from EdgeVPN.
+5. Node B performs a lookup in the shared ledger.
+6. If a match is found, Node B routes the packet back to the TUN interface, up to the application level.
+
+### Controller
+
+A set of Kubernetes Native Extensions ([Entangle](/docs/reference/entangle)) provides peer-to-peer functionalities also to existing clusters by allowing to bridge connection with the same design architecture described above.
+
+It can be used to:
+
+- Bridge services between clusters
+- Bridge external connections to cluster
+- Setup EdgeVPN as a daemonset between cluster nodes
+
+See also the Entangle [documentation](/docs/reference/entangle) to learn more about it.
+
+## Benefits
+
+<p align="center">
+<img width="700" src="https://user-images.githubusercontent.com/2420543/195459436-236139cf-605d-4608-9018-ea80381d4e77.png">
+</p>
+
+The use of p2p technology to enable self-coordination of Kubernetes clusters in Kairos offers a number of benefits:
+
+1. **Simplified deployment**: With Kairos, deploying Kubernetes clusters at the edge is greatly simplified. Users don’t need to specify any network settings or use a control management interface to set up and manage their clusters.
```

Comment:
I think it's ok to remove the "with Kairos" here and on other items since the line above mentions "Kairos offers"